Gemfile

```diff
@@ -0,0 +1,8 @@
+source "https://rubygems.org"
+
+gem "sass", "~> 3.3.14"
```

Comment:
As you are starting this fresh I'd suggest trying [node-sass](https://github.com/sass/node-sass) via [grunt-sass](https://github.com/sindresorhus/grunt-sass) rather than the default ruby sass as it uses the latest `libsass` which get much better compile times, and is the recommended version now.